R/load_data.R

```diff
@@ -10,8 +10,18 @@ run.demo <- function() {
   cat("Select snp data","\n")
   snp.file <- file.choose()
   cat("Read snp data","\n")
-  #In the SNP-array file only column 3,4 and 6 are of interest
-  snpm.data <- read.table(snp.file, skip = 9, header = T, sep = "\t")[,c(3,4,6)]
+
+  #In the SNP-array file only columns 'Chr', 'Position and 'B Allele Freq are of interest
+  #First find row with headers (following) row with '[Data]' in column 1.
+  snp.headerlines <- readLines( snp.file, n=20)
```

Comment:
20 seems fine , but is it based on some file standard ?  should it look further is no Data tag is found ?

Reply:
This is a good idea. Do you know how we could implement this?